src/ncurses_ui.cc

```diff
@@ -933,12 +954,13 @@ void NCursesUI::info_show(StringView title, StringView content,
     m_info.face = face;
     m_info.style = style;
 
-    const Rect rect = {m_status_on_top ? 1_line : 0_line, m_dimensions};
+    auto main_line_pos = get_zone_line("main");
+    const Rect rect = {{main_line_pos}, m_dimensions};
     InfoBox info_box;
     if (style == InfoStyle::Prompt)
     {
         info_box = make_info_box(m_info.title, m_info.content, m_dimensions.column, m_assistant);
-        anchor = DisplayCoord{m_status_on_top ? 0 : m_dimensions.line, m_dimensions.column-1};
+        anchor = DisplayCoord{m_status_on_top ? main_line_pos-1 : main_line_pos + m_dimensions.line, m_dimensions.column-1};
```

Comment:
I dont get why we use `m_status_on_top` here, the `get_zone_line` should give use the correct line offset.